src/services/ivaDatabaseService.ts

```diff
@@ -0,0 +1,66 @@
+import DynamoDB, { DocumentClient } from "aws-sdk/clients/dynamodb";
+import { IDBConfig } from "../models";
+import { Configuration } from "../utils/Configuration";
+import { PromiseResult } from "aws-sdk/lib/request";
+import {
+  EUVehicleCategory,
+  InspectionType,
+  VehicleType,
+} from "@dvsa/cvs-type-definitions/types/iva/defects/get";
+
+export class IvaDatabaseService {
```

Comment:
Because this is interacting with a db, might be good to have some error handling somewhere

Reply:
The error handling is already in the service layer.